docs/configuration.md

```diff
@@ -40,6 +40,7 @@ LMS. Each of these is mandatory to get the service working correctly.
 | `H_API_URL_PRIVATE`               | `https://cloud.hosting.url/api`        | URL for service to service communication        |
 | `H_API_URL_PUBLIC`                | `https://fr.hypothes.is/api`           | URL for client to service communication         |
 | `H_AUTHORITY`                     | `lms.eg.hypothes.is`                   | An authority to separate LMS annotations in `h` |
+| `REGION_CODE`                     | `us`                                   | The deployment's region code                    |
```

Comment:
I don't really know what these region codes are used for (something in report) so there's not much more I can say here.